sources/pip-req.txt

```diff
@@ -0,0 +1,54 @@
+argparse==1.2.1
```

Comment:
It's missing a lot of requirements appearently. This is why those guys 
created their own version
Sébastien Maillard --
*Sébastien Maillard* /[Technical Analyst]/
Phone: + 86 186 2103 7589 (Cell), + 86 21 6211 8017/27/37 (Office)
*Elico Shanghai* (Shenzhen/Singapore) <http://www.elico-corp.com> /*Odoo 
Gold Partner, best Odoo Partner 2014 for APAC*/
On 09/25/2015 05:41 PM, Augustin Cisterne-Kaas wrote:
>
> In sources/pip-req.txt 
> <https://github.com/Elico-Corp/odoo-docker/pull/4#discussion_r40414266>:
>
> > @@ -0,0 +1,54 @@
> > +argparse==1.2.1
>
> Should you use the |requirements.txt| from the |OCB/requirements.txt|?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Elico-Corp/odoo-docker/pull/4/files#r40414266>.
>